dev-requirements-py3.txt

```diff
@@ -1,7 +1,7 @@
 flake8==3.7.9
 bandit==1.6.2
 demisto-py==2.0.9
-demisto-sdk==1.0.0
+demisto-sdk==1.0.1
```

Comment:
should this bee here

Reply:
I'm not sure. I know we were updating the build to use the new SDK.

I'll check.

Reply:
It's 1.0.1 now but will be bumped to 1.0.2 today.